### ISSUE TITLE
USAToday

### ISSUE THREAD

**Issue by Grrrrandpa** [kebernet/shortyz#133]
USAToday puzzle last downloaded on March 31.  Since then, it does not.  Any ideas?

**Comment by frunction** [comment]
Yes, please check on this.   USA Today is the only pizzle I usually do.

**Comment by kebernet** [comment]
Updates here:

https://plus.google.com/b/112662859624418565027/112662859624418565027/posts/Fso9AMUfArd

I will be able to get it back, but it is going to take me a few days.

**Comment by Grrrrandpa** [comment]
USA Today is back.  IT'S BACK!!!!

Thank you, kind sir.

Grrrrandpa
